CHANGELOG.md

```diff
@@ -1,6 +1,20 @@
 <!-- markdownlint-disable MD024 -->
 # Changelog
 
+## `v0.7.0` (_aka_ 🦒)
+
+### Breaking changes
+
+In [#42](https://github.com/algorand/graviton/pull/42):
+
+* Inside `graviton/blackbox.py`: Removing the parameters `abi_argument_types` and `abi_return_type` from dry run execution methods, in favor of the unified `abi_method_signature` parameter. Also adding `omit_method_selector` and `validation` parameters to allow running non-ARC-4 compliant teal code that use ABI types.
+* Deleting all code previously marked with the `deprecated_` (or `Deprecated`) prefix. This includes:
+  * `graviton/deprecated_dryrun.py`
+  * `graviton/deprecated_dryrun_mixin.py`
+  * `tests/integration/dryrun_mixin_docs_test.py`
+* Deleting method `Invariant.inputs_and_invariants()`
```

Comment:
This is a good idea. But I will not touch until there's consensus about whether this PR gets merged first, or the child PR  #44 gets merged in before the merge to `main`.

Reply:
@tzaffi How does #44 affect the discussion here?

* I'll ultimately follow your lead.  I'm asking because I briefly scanned #44 and I don't immediately see the connection.
* I don't know enough about #44 yet to have an opinion on keeping #42 independent or not.  In case it helps inform your judgment - from my brief scan of #44, I suspect I'll need ~2 hours to make an initial pass.

Reply:
I think it is clear now that #44 will not be mergeable until after the hackathon. Therefore, I will make all the changes in _this PR_. The connection was purely exists because there are other similar concerns in that PR and it would be useful to bundle such changes. But it's not much harder to sequence such changes as well, and that's what I'll do.

Reply:
addressed in: https://github.com/algorand/graviton/pull/42/commits/0412b0d1a5b77619ce4d9b9ebd57cc2edd641aac

Reply:
@tzaffi Thanks for bringing in CHANGELOG feedback - resolving.